### PR TITLE
HMAN- 72 & HMAN-73

### PR DESCRIPTION
**Pull request by padmapriyanalam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-72
https://tools.hmcts.net/jira/browse/HMAN-73

### Change description ###
Validate Schema for GET "/unNotifiedHearings" request
Create search query for GET unNotifiedHearings response

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    @Override
+    public UnNotifiedHearingsResponse getUnNotifiedHearings(String hmctsServiceCode, LocalDateTime hearingStartDateFrom,
+                                                            LocalDateTime hearingStartDateTo) {
+        isValidHmctsServiceCode(hmctsServiceCode);
+        Page<Long> page = getUnNotifiedHearingResults(
+            hmctsServiceCode, hearingStartDateFrom, hearingStartDateTo);
+        List<String> hearingIds = getHearingIdInStrings(page.getContent());
+        UnNotifiedHearingsResponse response = getUnNotifiedHearingsResponse(hearingIds, page.getTotalElements());
+        return response;
```
Comment (RebeccaBaker) [inline src/main/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceImpl.java:47]:
```suggestion
        return getUnNotifiedHearingsResponse(hearingIds, page.getTotalElements());
```

File: src/main/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    private List<String> getHearingIdInStrings(List<Long> hearingIdsLong) {
+        List<String> hearingIds = hearingIdsLong.stream().map(Object::toString)
+            .collect(Collectors.toList());
+        return hearingIds;
```
Comment (RebeccaBaker) [inline src/main/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceImpl.java:53]:
```suggestion
        return hearingIdsLong.stream().map(Object::toString)
            .collect(Collectors.toList());
```

**Review by RebeccaBaker — APPROVED** [review]
approved with some minor comments

File: src/main/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceImpl.java (hunk 1 above)
Comment (padmapriyanalam) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceImpl.java]: changed

**Pull request merged**